src/azure-cli-core/azure/cli/core/auth/credential_adaptor.py

```diff
@@ -13,59 +13,33 @@
 
 
 class CredentialAdaptor:
-    def __init__(self, credential, resource=None, auxiliary_credentials=None):
-        """
-        Adaptor to both
-        - Track 1: msrest.authentication.Authentication, which exposes signed_session
-        - Track 2: azure.core.credentials.TokenCredential, which exposes get_token
+    def __init__(self, credential, auxiliary_credentials=None):
+        """Cross-tenant credential adaptor. It takes a main credential and auxiliary credentials.
+
+        It implements Track 2 SDK's azure.core.credentials.TokenCredential by exposing get_token.
```

Comment:
Even though Track 1 SDK auth support is dropped, `CredentialAdaptor` is preserved as it will be repurposed as an SDK-MSAL adaptor in https://github.com/Azure/azure-cli/pull/29955.